src/App.js

```diff
@@ -1,13 +1,51 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 1528817084358,
+          completed: false
+        }
+      ],
+      task: ''
+    };
+
+  }
+
+  addTodo = () => {
+    
+    const newTodos = [...this.state.todos]
+
+    newTodos.push({task: this.state.task, id: Date.now(), completed: false})
+    this.setState({todos: newTodos, task: ''})
+  }
+
+  handleChanges = e => {
```

Comment:
```suggestion
 handleChange = e => {
		const { type, name, value } = e.target;
		const val = type === 'number' ? parseFloat(value) : value;
		this.setState({ [name]: val });
	};
```

Reply:
a more verbose handleChange method that also converts any number represented as a string to an actual number.